src/DevSoc.Eventathon.Attendance/AttendanceService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Dapper;
+using DevSoc.Eventathon.Attendance;
+using DevSoc.Eventathon.Calendars.Models;
+
+namespace DevSoc.Eventathon.Data;
+
+public class AttendanceService : IAttendanceService
```

Comment:
This is the data layer, so it's the repository pattern - rename to `AttendanceRepository` / `IAttendanceRepository`

Reply:
I can't do that as dependency injection requires them to be in the IAttendanceService format

Reply:
Nevermind, I think I was just doing something wrong